pkg/config/validator.go

```diff
@@ -98,8 +115,14 @@ type (
 	validator struct {
 		managerFinder    AspectValidatorFinder
 		adapterFinder    BuilderValidatorFinder
+		bldrInfoFinder   BuilderInfoFinder
+		setupHandler     SetupHandlerFn
+		tmplRepo         template.Repository
 		findAspects      AdapterToAspectMapper
 		descriptorFinder descriptor.Finder
+		hndlrBldrByName  map[string]*HandlerBuilderInfo
+		cnstrByName      map[string]*pb.Constructor
```

Comment:
I think `ctor` is a better/more widely used abbreviation. `cnstr` looks too much like `const`. I'd call this field `ctors`